### PR TITLE
fix the top navigation

### PR DESCRIPTION
**Pull request by herenacreative**
Screen Shot 2023-05-30 at 10 22 58

localhost_3000__tab=register(iPhone 12 Pro) (1)

### CODE REVIEW

File: components/header/index.tsx [hunk 1]
```diff
@@ -61,11 +61,11 @@ export default function Header({
                 </a>
               </li>
               <li className="nav-item">
-                <Link
+                <a
```
Comment (nicnocquee) [inline components/header/index.tsx]: Please keep using Link

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: components/header/index.tsx (hunk 1 above)
Comment (herenacreative) [reply components/header/index.tsx]: updated

**Comment by raosan** [comment]
closing because new fix is simpler #10

**Comment by raosan** [comment]
closing because new fix is simpler #10

**Pull request closed without merging**